### DISCUSSION TITLE
Please make dependabot alers exportable  to cvs

### DISCUSSION THREAD

**Discussion by ngamboa** [community/community#73162] [Enterprise]
### Select Topic Area

Product Feedback

### Body

Since we started using GHE, its has been nothing short of a headache to try to get any data out of the system for executive reporting, it would be so simple for GHE to just provide a download link along with results when you set up any kind of filter search.  Alas, there is nothing in the UI that supports this so its a blocker for us manager types who do not work with API's yet we do the reporting.

**Comment by kellyarwine** [comment]
Hi, @ngamboa! Thank you for the feedback! I'm one of the product managers for security products. I have logged your user feedback for CSV exporting for consideration.

Related, today we have a launch that might help you, check [this page](https://github.blog/changelog/label/advanced-security/) later today for details.

Also, did you know you can use the [API](https://docs.github.com/en/rest/dependabot/alerts?apiVersion=2022-11-28) to get the alerts? That might be an option for you.

**Comment by balajibhai** [comment]
Hi @kellyarwine, Is there any way to download as a whole executive summary report (not a detailed report) which contains all the security vulnerabilities/alerts from dependabot, code scanning, secret scanning as a csv format other than using API? Also if there are any plugins please suggest. Thank you!

**Reply by kellyarwine** [reply]
> which contains all the security vulnerabilities/alerts from dependabot, code scanning, secret scanning as a CSV format other than using API?

_[deleted]_

I'll circle back and delete my details in the next 3 days to avoid spam.

**Reply by balajibhai** [reply]
Thank you for the response! I will email you.
